sklearn/preprocessing/data.py

```diff
@@ -2865,51 +2866,60 @@ def power_transform(X, method='box-cox', standardize=True, copy=True):
     Parameters
     ----------
     X : array-like, shape (n_samples, n_features)
-        The data to be transformed using a power transformation.
+        The data used to estimate the optimal transformation parameters.
 
-    method : str, (default='box-cox')
-        The power transform method. Currently, 'box-cox' (Box-Cox transform)
-        is the only option available.
+    method : str, (default='yeo-johnson')
+        The power transform method. Available methods are:
+
+        - 'yeo-johnson' [1]_, works with positive and negative values
+        - 'box-cox' [2]_, only works with strictly positive values
 
     standardize : boolean, default=True
         Set to True to apply zero-mean, unit-variance normalization to the
         transformed output.
 
     copy : boolean, optional, default=True
-        Set to False to perform inplace computation.
+        Set to False to perform inplace computation during transformation.
 
     Examples
```

Comment:
Just noticed that the docstring is missing a `Return` section

Reply:
added